mathics/core/rules.py

```diff
@@ -232,9 +232,9 @@ def __getstate__(self):
         return odict
 
     def __setstate__(self, dict):
-        from mathics.builtin import builtins
+        from mathics.builtin import _builtins
 
         self.__dict__.update(dict)  # update attributes
-        cls, name = dict["function_"]
+        class_name, name = dict["function_"]
 
-        self.function = getattr(builtins[cls], name)
+        self.function = getattr(_builtins[class_name], name)
```

Comment:
I Just wonder, is it OK to keep the name ``_builtins`` for something that is not private to the module?

Reply:
This is a temporary change. #635 or a stripped down version of that has the more complete fix.

Reply:
In that case, let's merge and iterate.